codecov_auth/models.py

```diff
@@ -471,6 +475,48 @@ def remove_admin(self, user):
         self.save()
 
 
+GITHUB_APP_INSTALLATION_DEFAULT_NAME = "codecov_app_installation"
+
+
+class GithubAppInstallation(
```

Comment:
do we also need an app id column? since we may use sentry's app or support multiple apps

Reply:
the "installation_id" is the "app_id"